lib/new_relic/agent/instrumentation/ruby_openai/instrumentation.rb

```diff
@@ -30,6 +30,7 @@ def embeddings_instrumentation(parameters)
       segment = NewRelic::Agent::Tracer.start_segment(name: EMBEDDINGS_SEGMENT_NAME)
       record_openai_metric
       event = create_embeddings_event(parameters)
+      event.remove_instance_variable(:@input) if !record_content_enabled?
```

Comment:
I'm torn between making this a method similar to `def remove_content(messages)` for Message events. The code is small so I left it—but open to creating a method for cleaner consistency.